longshot/main.tf

```diff
@@ -0,0 +1,24 @@
+variable "papertrail_prod_destination" {}
+variable "papertrail_qa_destination" {}
+
+resource "heroku_pipeline" "longshot" {
+  name = "longshot"
+}
+
+module "longshot-qa" {
+  source = "application"
+
+  name           = "longshot-qa"
+  host           = "longshot-qa.dosomething.org"
+  pipeline       = "${heroku_pipeline.longshot.id}"
+  pipeline_stage = "staging"
+
+  email_name    = "Longshot QA"
+  email_address = "devops@dosomething.org"
+
+  database_name  = "whitelabel_scholarship"
+  database_type  = "db.t2.micro"
+  database_scale = 5
```

Comment:
Is `database_scale` a Terraformism, or can we control the name of this var?

Reply:
For clarity, the name seems unclear at a glance, and something like `database_storage_size` or something like that would be clearer, I think.

Reply:
That's a fair point! I'd just used `scale` for consistency with other variables here.

Reply:
Renamed `database_scale` to `database_size` and added a description to clarify!